plugins/tailwind/compiler.ts

```diff
@@ -62,6 +70,31 @@ export async function initTailwind(
     return pattern;
   });
 
+  const imports = (await import(path.toFileUrl(config.denoJsonPath).href, {
+    with: { type: "json" },
+  })).default;
+  for (const plugin of config.plugins ?? []) {
+    if (!plugin.location) continue;
+    // if the plugin is declared in a separate place than the project, the plugin developer should have specified a projectLocation
+    // otherwise, we assume the plugin is in the same directory as the project
+    const projectLocation = plugin.projectLocation ??
+      path.dirname(plugin.location);
+    const resolvedImports = await parseFromJson(
+      path.toFileUrl(config.denoJsonPath),
+      imports,
+    );
+
+    const moduleGraph = await createGraph(plugin.location, {
+      resolve: createCustomResolver(resolvedImports),
+    });
```

Comment:
```suggestion
    const { resolve } = await parseFromJson(
      path.toFileUrl(config.denoJsonPath),
      imports,
    );

    const moduleGraph = await createGraph(plugin.location, { resolve });
```
`createCustomResolver()` may not be needed.

Reply:
Why? This is the key part of the code to help us identify files that are part of the plugin.

Reply:
To help clarify: I need fully resolved URLs so that I can fetch them later.

Reply:
Oh, I see your comment on deno_graph now. I'll play around with it more later, but the suggestion here doesn't immediately work.

Reply:
```ts
    const resolvedImports = await parseFromJson(
      path.toFileUrl(config.denoJsonPath),
      imports,
    );

    const moduleGraph = await createGraph(plugin.location, {
      resolve: resolvedImports.resolve.bind(resolvedImports),
    });
```
ended up working. thanks